contracts/sol/IncrementalMerkleTree.sol

```diff
@@ -19,11 +19,14 @@
  * along with Semaphore.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-pragma solidity ^0.5.0;
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.3;
+
+import "./SnarkConstants.sol";
+import "./Hasher.sol";
+import "../node_modules/@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
Imports like this may not work for projects which use `maci-contracts` package because `@openzeppelin` can be in different location.